tests/specs/happyPaths.shelley.15.test.ts

```diff
@@ -0,0 +1,138 @@
+import * as addWalletsScreen from '../screenObjects/addWallets.screen'
+import * as addWalletScreen from '../screenObjects/addWallet.screen'
+import * as createNewWalletCredentialsScreen from '../screenObjects/createWalletScreens/createWalletCredentials.screen'
+import * as nobodyLookingScreen from '../screenObjects/createWalletScreens/nobodyLookingNotification.screen'
+import * as recoveryPhraseRememberScreen from '../screenObjects/createWalletScreens/recoveryPhraseRemember.screen'
+import * as recoveryPhraseNotificationScreen from '../screenObjects/createWalletScreens/recoveryPhraseNotification.screen'
+import * as recoveryPhraseEnterScreen from '../screenObjects/createWalletScreens/recoveryPhraseEnter.screen'
+import * as myWalletsScreen from '../screenObjects/myWallets.screen'
+import * as walletHistoryScreen from '../screenObjects/walletHistory.screen'
+import * as sendScreen from '../screenObjects/send.screen'
+import {
+  checkForErrors,
+  enterNewValue,
+  enterPinCodeIfNecessary,
+  enterRecoveryPhrase,
+  enterWalletCredentials,
+  firstAppLaunch,
+  hideKeyboard,
+  prepareIntrawalletTx,
+} from '../helpers/utils'
+import {
+  WALLET_NAME,
+  DEFAULT_TIMEOUT,
+  DEFAULT_INTERVAL,
+  VALID_PIN,
+  NORMAL_15_WORD_WALLET,
+  WalletType,
+  SPENDING_PASSWORD,
+} from '../constants'
+import * as selectWalletToRestoreScreen from '../screenObjects/selectWalletToRestore.screen'
+import * as recoveryPhraseInputScreen from '../screenObjects/restoreWalletsScreens/recoveryPhraseEnterManually.screen'
+import * as verifyRestoredWalletScreen from '../screenObjects/restoreWalletsScreens/verifyRestoredWallet.screen'
+
+const expect = require('chai').expect
+
+describe('Happy paths', () => {
+  // Execute a block of code before every tests
+  beforeEach(() => {
+    driver.launchApp()
+  })
+  // Execute a block of code after every tests
+  afterEach(() => {
+    driver.closeApp()
+  })
+
+  it('Prepare the app', async () => {
+    await firstAppLaunch(VALID_PIN)
```

Comment:
It will require the PIN only after having at least 1 wallet created.

Reply:
The behavior  is changed in the next [PR](https://github.com/Emurgo/yoroi-mobile/pull/2098/files)